beacon_node/network/src/network_beacon_processor/mod.rs

```diff
@@ -775,6 +777,69 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
                 "error" => %e)
         });
     }
+
+    /// Re-seed the network with reconstructed data columns.
+    fn handle_data_columns_to_publish(
```

Comment:
Maybe just inline below now that this function is quite simple

Reply:
Done